.github/workflows/tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: Python 3.11 Workflow
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.11"
+
+    - name: Run Python scripts and check output
+      run: |
+        output=$(python *.py)
```

Comment:
This will run all python files, do we want to do this? Or provide them with specific file names?